src/lib/operators/operator_performance_data.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+#include "types.hpp"
+
+namespace opossum {
+
+// For an example on how this can be extended on a per-operator basis, see JoinIndex
+
+struct OperatorPerformanceData : public Noncopyable {
+  virtual ~OperatorPerformanceData();
+
+  std::chrono::microseconds walltime{0};
```

Comment:
Does it make sense to store the wall time in microseconds when we print nanoseconds later?

Reply:
That's what we get from the `Timer` class. We don't print ns but convert it into the most readable format. ns is just for the interface of `format_duration`.